src/idb.filesystem.js

```diff
@@ -34,16 +34,17 @@ if (exports.requestFileSystem || exports.webkitRequestFileSystem) {
 }
 
 // Bomb out if no indexedDB available
-const indexedDB = exports.indexedDB || exports.mozIndexedDB ||
+var indexedDB = exports.indexedDB || exports.mozIndexedDB ||
                   exports.msIndexedDB;
 if (!indexedDB) {
   return;
 }
 
-let IDB_SUPPORTS_BLOB = true;
+var IDB_SUPPORTS_BLOB = false;
```

Comment:
Most modern browsers support let/const out of the box and it's only a matter of time before Safari 10 supports both.  I'd also like to move to full es6 support eventually so it's more important that the untransformed source be easier to maintain.